lib/tabs/KTabsList.vue

```diff
@@ -0,0 +1,324 @@
+<template>
+
+  <div
+    v-show="enablePrint || !$isPrint"
+    role="tablist"
+    :[ariaLabelAttrName]="ariaLabelAttr"
+  >
+    <template v-if="useRouter">
+      <!-- https://v3.router.vuejs.org/api/#example-applying-active-class-to-outer-element -->
```

Comment:
thanks for including this

Reply:
Happy it's helpful for you too. I will certainly need it as I will forget soon, I think I actually didn't know about the `custom` router link before this work.